### ISSUE TITLE
Prom bundle configuration should take registry as an option.

### ISSUE THREAD

**Issue by aaronleesmith** [jochen-schweizer/express-prom-bundle#45]
Currently, this package always uses the ES6 global import of `client.register` to register metrics (the default). It should also support custom registries as part of the configuration. This is very helpful in testing and larger apps where multiple registries are used.

**Comment by poteirard** [comment]
I have the same problem. I think this line is wrong:

```ts
// /types/index.d.ts
declare function express_prom_bundle(opts: express_prom_bundle.Opts): RequestHandler;
```

It should be something like:

```ts
declare function express_prom_bundle(opts: express_prom_bundle.
{ 
	metrics: ...
	metricsMiddleware: RequestHandler,
	promClient: ...
};
```

Maybe @raszi can put a bit of light on this? I would be happy to help with a PR

**Comment by disjunction** [comment]
fixed in #46
